### ISSUE TITLE
Sync 2 images

### ISSUE THREAD

**Issue by jorgemedina2406** [google/marzipano#433]
Hello how are you? I hope you can help me here. I am opening 2 images side by side, both move separately, what I want is to add a button that when clicked the images start to move together, that is, if I move image 1, image 2 also moves, or vice versa, and if I click the button again, they move separately again.

**Comment by chmelevskij** [comment]
you mean something like https://www.marzipano.net/demos/side-by-side/ ?

which has example here https://github.com/google/marzipano/tree/master/demos/side-by-side

**Comment by jorgemedina2406** [comment]
@chmelevskij Yes, but I don't quite understand the part where it does this: {z}/{f}/{y}/{x}.jpg I don't know what that is, I only have the 360 images.

**Comment by chmelevskij** [comment]
`{z}/{f}/{y}/{x}.jpg` is the naming convention for cubemap tiles.

What you probably have is called `Equirectangular` image. https://github.com/google/marzipano/blob/master/demos/equirect/index.js you can use this demo as an example here.

**Comment by jorgemedina2406** [comment]
@chmelevskij Thank you very much for your answer, if I do it like this, I can show the 2 images perfectly, they move separately, but wanting to move the 2 images at the same time after clicking on a button is what I still can't.

**Comment by campmdabt** [comment]
@jorgemedina2406 You need to start with the code in the side-by-side demo, then change the Marzipano Source to use the format shown in the equirectanuglar demo instead of the cubemap type Source used in the side-by-side demo.
